### PR TITLE
Bootstrap basic example contents for 1.0

### PR DESCRIPTION
**Pull request by benjaoming**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,83 @@
+Example: Basic Sphinx project for Read the Docs
```
Comment (ericholscher) [inline README.rst:1]: Guessing this file meant to be in this PR?

**Review by ericholscher — APPROVED** [review]
Simple enough 👍

File: README.rst (hunk 1 above)
Comment (benjaoming) [reply README.rst:1]: Ah sorry no - I was just using it to copy from, as mkdocs is Markdown-only, we can use a README.md.

**Comment by benjaoming** [comment]
@ericholscher thanks for the review!

tl;dr This was a quick draft to open the arena on the main question here: Do we want a "basic" mkdocs example WITH API docs or not? I think it's great to use an example *with* API docs since it shows the big benefits of having code and documentation live alongside and with the same release lifecycle + reflects the basic sphinx example projects pretty much 1:1.

I understand, though, that there are some strong feelings of why MkDocs should not even support API documentation as a core thing, and that some people just think that we are better off without auto-generated docs. I can find the reference later. Yet, I think that we can make a choice to say that this *basic* example will anticipate API docs as a *basic* component.

**Review by benjaoming — COMMENTED** [review]
This is now pretty much where I would see it for a 1.0 -- I never managed to make highlight.js work locally for the mkdocs readthedocs theme, was it was advertised to. Maybe it'll work on Read the Docs :shrug: 

It's interesting that even for a basic example, I would argue that MkDocs needs extensions. However, on the bright side, there's a lot of those.
